src/pages/Steps.tsx

```diff
@@ -92,33 +97,39 @@ type StepProps<TFormData extends FormDataIntersection> = Omit<
   onChange: (value: any) => void;
 };
 
-type StepsProps<
-  TFormData extends FormDataIntersection
-> = UseFormReturn<TFormData> & {
-  mode: 'UPDATE' | 'CREATE';
+type StepsProps<TFormData extends FormDataIntersection> = {
+  eventId?: string;
+  form: UseFormReturn<TFormData>;
   fieldLoading?: string;
   onChange?: (value: string, field: string) => void;
+  onChangeSuccess?: (editedField: string) => void;
   configuration: StepsConfiguration<TFormData>;
 };
 
 const Steps = <TFormData extends FormDataIntersection>({
-  mode,
   onChange,
   configuration,
   fieldLoading,
+  form,
+  eventId,
   ...props
 }: StepsProps<TFormData>) => {
-  const keys = Object.keys(props.getValues());
+  const keys = Object.keys(form.getValues());
+
+  const mode = eventId ? 'UPDATE' : 'CREATE';
```

Comment:
If I'm not mistaken this mode isn't used anywhere

Reply:
https://github.com/cultuurnet/udb3-frontend/blob/pass-down-form-to-step-configuration/src/pages/Steps.tsx#L137